app/dashboard/models.py

```diff
@@ -55,6 +55,40 @@
 logger = logging.getLogger(__name__)
 
 
+class BountyRequest(SuperModel):
+    """Define the structure of a Request."""
+
+    STATUS_FUNDED = 'funded'
+    STATUS_UNFUNDED = 'unfunded'
+
+    REQUEST_STATUSES = (
+        (STATUS_FUNDED, 'funded'),
+        (STATUS_UNFUNDED, 'unfunded'),
+    )
+
+    github_url = models.CharField(max_length=255, blank=True)
+    amount = models.DecimalField(null=True, blank=True, decimal_places=2, max_digits=50)
+    denomination = models.CharField(default='ETH', max_length=5)
+    creator = models.ForeignKey('dashboard.Profile', related_name='bounty_requests', null=True, on_delete=models.SET_NULL)
+    status = models.CharField(max_length=8, choices=REQUEST_STATUSES, default=STATUS_FUNDED)
+    followers = models.ManyToManyField('dashboard.Profile')
+    bounty = models.ForeignKey('dashboard.Bounty', related_name='requests', null=True, on_delete=models.SET_NULL)
+
+    def __str__(self):
+        return self.github_url
+
+
+class Comment(models.Model):
```

Comment:
Can we rename this to avoid future collision with the django comments package and subclass `SuperModel` from `economy.models` ?  Then you can drop `created_on` from the below definition.